app/__init__.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask

```

Comment:
Module-level docstring is a good practice.
like """Application module bla bla."""